storage/rocksdb/ha_rocksdb.cc

```diff
@@ -4574,6 +4584,17 @@ int ha_rocksdb::convert_record_to_storage_format(
   *packed_rec =
       rocksdb::Slice(m_storage_record.ptr(), m_storage_record.length());
 
+  // We've obtained the TTL timestamp for this record. Now store it in
+  // m_ttl_bytes if it wasn't already so we can update secondary indexes with
+  // this value as well.
+  if (has_ttl) {
+    if (!row_info.old_pk_slice.empty()) {
+      m_ttl_bytes_updated = memcmp(m_ttl_bytes, m_storage_record.ptr(),
+                                   ROCKSDB_SIZEOF_TTL_RECORD);
```

Comment:
i think you can optimize further here by also checking if there is a `ttl_column`.  Because without a ttl column, you are guaranteed that the timestamp remains the same.  So now you can skip this memcmp.